src/Graby.php

```diff
@@ -43,7 +43,7 @@ public function __construct($config = [], Client $client = null, ConfigBuilder $
     {
         $resolver = new OptionsResolver();
         $resolver->setDefaults([
-            'debug' => false,
```

Comment:
Removing a configuration mean it'll break backward compatibility.
Why don't you want to keep `debug`?
- debug = true, we want verbose log
- debug = false, we don't want verbose log

Or if you want to keep different log, introduce a new log level configuration, with default value to "info" and when it's "debug", give more log.

Reply:
I am not sure I understand what you mean. I get it about backward compatibility, but what would you want instead?

The issue with keeping just `debug = true/false` is that I cannot introduce the `Logger::INFO` level easily in the if/else statement. In the current situation, when `debug = true`, the equivalent to my `log_level = info` is logged in graby.log ; else nothing is logged with `debug = false`.

I think that most people won't use the "full debug", so I wouldn't want to activate it by default and give the choice between the current debug mode and the more detailed one. So I'd need a way to still use two levels of Logger, with one variable. Do you have an idea in mind?

Reply:
Cf commit https://github.com/j0k3r/graby/pull/184/commits/eca074a8029860119ccbdc3642029866a84ff5b5